src/cltk/alphabet/lat.py

```diff
@@ -1,12 +1,471 @@
-"""Alphabet and text normalization for Latin."""
+"""Alphabet and text normalization for Latin.
+
+- *Principles of Text Cleaning gleaned from*
+- http://udallasclassics.org/wp-content/uploads/maurer_files/APPARATUSABBREVIATIONS.pdf
+
+Guidelines:
+- [...] Square brackets, or in recent editions wavy brackets ʺ{...}ʺ, enclose words etc. that an editor thinks should be deleted (see ʺdel.ʺ) or marked as out of place (see ʺsecl.ʺ).
+- [...] Square brackets in a papyrus text, or in an inscription, enclose places where words have been lost through physical damage.
+- If this happens in mid-line, editors use ʺ[...]ʺ.
+- If only the end of the line is missing, they use a single bracket ʺ[...ʺ
+- If the lineʹs beginning is missing, they use ʺ...]ʺ
+- Within the brackets, often each dot represents one missing letter.
+- [[...]] Double brackets enclose letters or words deleted by the medieval copyist himself.
+- (...) Round brackets are used to supplement words abbreviated by the original copyist; e.g. in an inscription: ʺtrib(unus) mil(itum) leg(ionis) IIIʺ
+- <...> diamond ( = elbow = angular) brackets enclose words etc. that an editor has added (see ʺsuppl.ʺ)
+- †  An obelus (pl. obeli) means that the word(s etc.) is very plainly corrrupt, but the editor cannot see how to emend.
+- If only one word is corrupt, there is only one obelus, which precedes the word; if two or more words are corrupt, two obeli enclose them. (Such at least is the rule--but that rule is often broken, especially in older editions, which sometimes dagger several words using only one obelus.) To dagger words in this way is to ʺobelizeʺ them.
+
+"""
+__author__ = [
+    "Todd Cook <todd.g.cook@gmail.com>",
+    "Kyle P. Johnson <kyle@kyle-p-johnson.com>",
+]
+__license__ = "MIT License"
+
+import re
+from typing import Dict
 
 from cltk.alphabet.text_normalization import cltk_normalize
 
+BRACE_STRIP = re.compile(r"{[^}]+}")
+NUMERALS = re.compile(r"[0-9]+")
+LATIN_PUNCT = re.compile(r"[\\/':;,!\?\._『@#\$%^&\*]+")
+QUOTES = re.compile(r'["”“]+')
+# we will extact content between brackets, it is editorial
+ANGLE_BRACKETS = re.compile(r"([a-zA-Z]+)?<[a-zA-Z\s]+>([,\?\.a-zA-Z]+)?")
+SQUARE_BRACKETS = re.compile(r"\[[^\]]+\]")
+OBELIZED_WORDS = re.compile(r"†[^†]+†")
+OBELIZED_WORD = re.compile(r"†[^\s]+\s")
+OBELIZED_PLUS_WORDS = re.compile(r"[\+][^\+]+[\+]")
+OBELIZED_PLUS_WORD = re.compile(r"[\+][^\s]+\s")
+HYPHENATED = re.compile(r"\s[^-]+-[^-]+\s")
```

Comment:
See my comment about de-hypenate. Maybe this regex already handles newlines? I would need to compile this myself to know, you can tell me if possible.

Reply:
So, my aim and intuition here is that we want to handle the majority of cases, but not do harm because we try to salvage sloppy editing. I'll look for some concrete example to vet the expansion, or spot where the code may be doing harm.

Reply:
> not do harm because we try to salvage sloppy editing
Good point. Resolving.